### DISCUSSION TITLE
Data gap count

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#101] [Q&A] [answered]
Hello,

We are running station position calculation for 13 years , but not have all the data for every day for this entire period. Does GROOPS have the capability to make a list with for example the days when there is no data available (like a Gap count with a time /date determination)? This would help o quickly see when there is no data in a time series if it is an error with the script or no data is actually available for that time. I saw a script called Sinex2StationDiscontinuities but i am not sure if this does do that.

Thanks for the help

Cheers
Jean

**Comment by tmayerguerr** [comment, marked as answer]
Hi,

Here are only some general ideas:

1. Generate a time series of you position files with [InstrumentConcatenate](https://groops-devs.github.io/groops/html/InstrumentConcatenate.html)
2. Generate a time series without gaps with [TimeSeriesCreate](https://groops-devs.github.io/groops/html/TimeSeriesCreate.html).
3. Remove from the second the first time series with [InstrumentRemoveEpochsByTimes](https://groops-devs.github.io/groops/html/InstrumentRemoveEpochsByTimes.html) (timePoints->instrument is the first time series).

Maybe you can also look at:
- [InstrumentConcatenate](https://groops-devs.github.io/groops/html/InstrumentConcatenate.html)
- [InstrumentSynchronize](https://groops-devs.github.io/groops/html/InstrumentSynchronize.html)
- [InstrumentArcStatistics](https://groops-devs.github.io/groops/html/InstrumentArcStatistics.html)
- Maybe [InstrumentArcCalculate](https://groops-devs.github.io/groops/html/InstrumentArcCalculate.html)  with the additional statistics file.

regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten,

Thanks for the quick feedback, I understand the logic in what the proposal is - basically do the entire timeseries, do the timeseries without
the gap and the difference of both are the gaps. I will also try some other examples you mentioned.

Txs for the advice

Cheers
Jean

**Comment by Bemtgenj** [comment]
Hi Torsten

the first solution you proposed worked perfectly

TS for the help

Cheers
Jean
